### PR TITLE
feat: Shape up docker image

### PR DESCRIPTION
**Pull request by chonla**
This PR shape up docker image by using node 18-slim as base image instead of node 18.

### CODE REVIEW

**Review by zkan — APPROVED** [review]
Nice one! Thanks krub @chonla

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:18
+FROM node:18-slim
```
Comment (kahnwong) [inline Dockerfile:1]: does it also work with `alpine` tag? alpine is lighter than slim.

Comment (zkan) [reply Dockerfile:1]: Oops. Could you please see the comment above? @chonla

Comment (chonla) [reply Dockerfile:1]: @kahnwong @zkan 
Thanks for feedback.

Comment (zkan) [reply Dockerfile:1]: FYI - https://github.com/thedataengineeringbook/thedataengineeringbook/pull/47

**Pull request merged**